src/containers/PodcastPage/PodcastSeriesListPage.tsx

```diff
@@ -114,69 +134,100 @@ const PodcastSeriesListPage = () => {
   }
 
-  if (loading) {
-    return <PageSpinner />;
-  }
-
   return (
     <>
       <HelmetWithTracker title={t("htmlTitles.podcast", { page: page })} />
-      <OneColumn>
-        <Heading element="h1" headingStyle="h1-resource" margin="xlarge">
-          {t("podcastPage.podcasts")}
-        </Heading>
-        {results?.length && results.length > 0 ? (
-          <>
-            <Heading element="h2" headingStyle="h2">
-              {t("podcastPage.subtitle")}
-            </Heading>
-            {results.map((series) => {
-              return <PodcastSeries key={`podcast-${series.id}`} {...series} />;
-            })}
-            <StyledPageNumber>{t("podcastPage.pageInfo", { page, lastPage })}</StyledPageNumber>
-          </>
-        ) : (
-          <NoResult>{t("podcastPage.noResults")}</NoResult>
-        )}
-        <StyledPaginationRoot
-          page={page}
-          onPageChange={(details) => onQueryPush({ ...searchObject, page: details.page })}
-          count={data?.podcastSeriesSearch?.totalCount ?? 0}
-          pageSize={pageSize}
-          translations={componentTranslations}
-          siblingCount={2}
-        >
-          <PaginationPrevTrigger asChild>
-            <Button variant="tertiary">
-              <ArrowLeftShortLine />
-              {t("pagination.prev")}
-            </Button>
-          </PaginationPrevTrigger>
-          <PaginationContext>
-            {(pagination) =>
-              pagination.pages.map((page, index) =>
-                page.type === "page" ? (
-                  <PaginationItem key={index} {...page} asChild>
-                    <Button variant={page.value === pagination.page ? "primary" : "tertiary"}>{page.value}</Button>
-                  </PaginationItem>
+      <main>
+        <Hero content="primary">
+          <HeroBackground />
+          <OneColumn>
+            <HeroContent>
+              <HomeBreadcrumb
+                items={[
+                  {
+                    name: t("breadcrumb.toFrontpage"),
+                    to: "/",
+                  },
+                  {
+                    name: t("podcastPage.podcasts"),
+                    to: "/podkast",
+                  },
+                ]}
+              />
+            </HeroContent>
+            <ArticleWrapper>
+              <ArticleHeader>
+                <TitleWrapper>
+                  <ContentTypeBadgeNew contentType={"podcast"} />
+                  <Heading id={SKIP_TO_CONTENT_ID} tabIndex={-1}>
+                    {t("podcastPage.podcasts")}
+                  </Heading>
+                  {!!results?.length && (
+                    <Heading asChild consumeCss textStyle="title.medium">
+                      <h2>{t("podcastPage.subtitle")}</h2>
+                    </Heading>
+                  )}
+                </TitleWrapper>
+              </ArticleHeader>
+              <ArticleContent asChild consumeCss>
+                {loading ? (
+                  <Spinner />
```

Comment:
Lagde ikke lars en `PageSpinner`?

Reply:
Jo, men den er for å vises når det ikke er annet innhold på siden, jeg syntes det var ryddig å kun skjule det som faktisk lastes ..  🤔